monitor.go

```diff
@@ -129,10 +130,10 @@ func newDockerClient() (*docker.Client, error) {
 		return docker.NewClient(dockerHost)
 	}
 
-	cert_path := os.Getenv("DOCKER_CERT_PATH")
-	ca := fmt.Sprintf("%s/ca.pem", cert_path)
-	cert := fmt.Sprintf("%s/cert.pem", cert_path)
-	key := fmt.Sprintf("%s/key.pem", cert_path)
+	certPath := os.Getenv("DOCKER_CERT_PATH")
```

Comment:
Please don't make other changes in the same commit; feel free to submit as a followup PR